solo-chains/runtime/starlight/src/genesis_config_presets.rs

```diff
@@ -267,70 +308,83 @@ fn starlight_testnet_genesis(
     const ENDOWMENT: u128 = 1_000_000 * STAR;
 
     serde_json::json!({
-        "balances": {
-            "balances": endowed_accounts.iter().map(|k| (k.clone(), ENDOWMENT)).collect::<Vec<_>>(),
-        },
-        "session": {
-            "keys": initial_authorities
-                .iter()
-                .map(|x| {
-                    (
-                        x.stash.clone(),
-                        x.stash.clone(),
-                        starlight_session_keys(
-                            x.babe.clone(),
-                            x.grandpa.clone(),
-                            x.para_validator.clone(),
-                            x.para_assignment.clone(),
-                            x.authority_discovery.clone(),
-                            x.beefy.clone(),
-                            x.nimbus.clone(),
-                        ),
-                    )
-                })
-                .collect::<Vec<_>>(),
-            "nonAuthorityKeys": invulnerables
-                .iter()
-                .cloned()
-                .map(|x| {
-                    (
-                        x.stash.clone(),
-                        x.stash.clone(),
-                        starlight_session_keys(
-                            x.babe.clone(),
-                            x.grandpa.clone(),
-                            x.para_validator.clone(),
-                            x.para_assignment.clone(),
-                            x.authority_discovery.clone(),
-                            x.beefy.clone(),
-                            x.nimbus.clone(),
-                        ),
-                    )
-                })
-                .collect::<Vec<_>>(),
-        },
-        "babe": {
-            "epochConfig": Some(BABE_GENESIS_EPOCH_CONFIG)
-        },
-        "sudo": { "key": Some(root_key.clone()) },
-        "configuration": {
-            "config": runtime_parachains::configuration::HostConfiguration {
-                scheduler_params: SchedulerParams {
-                    max_validators_per_core: Some(1),
-                    num_cores: para_ids.len() as u32,
-                    ..default_parachains_host_configuration().scheduler_params
-                },
-                ..default_parachains_host_configuration()
+    "balances": {
```

Comment:
Something messed up the formatting here. rustfmt doesn't work inside macros so not sure how to fix this.